ts/session/messages/outgoing/content/sync/SyncMessage.ts

```diff
@@ -12,5 +13,20 @@ export abstract class SyncMessage extends ContentMessage {
     });
   }
 
+  protected createSyncMessage(): SignalService.SyncMessage {
```

Comment:
wouldn't it be better to stick this code inside `syncProto()`?
The subclasses then can override it and add stuff to it

Reply:
Fixed